modules/test/playwright/tests/frontend-data-set-web/main/advanced.spec.ts

```diff
@@ -1209,9 +1201,27 @@ test(
 			expect(sentKeyValues).toEqual([]);
 			expect(sentSelectAll).toBe(true);
 
-			expect(await bulkActionsButton.getAttribute('aria-expanded')).toBe(
-				'false'
-			);
+			expect(
+				await fdsSamplePage.bulkActions.actionsDropdownButton.getAttribute(
+					'aria-expanded'
+				)
+			).toBe('false');
+
+			await fdsSamplePage.selectionToolbar.clearButton.click();
+		});
+
+		const firstListItem = fdsSamplePage.list.items.first();
+
+		const firstListItemCheckbox = firstListItem.getByRole('checkbox');
+
+		await test.step('Open list visualization mode', async () => {
+			await fdsSamplePage.changeVisualizationMode('List');
+		});
+
+		await test.step('Select the first item in the list', async () => {
+			await firstListItemCheckbox.check();
+
+			await expect(firstListItem).toHaveClass(/active/);
```

Comment:
Direct testing of computed styles is difficult, and overlaps with Clay functionality. I think it's best for us to only test correct CSS class setting, which is done in our code.